spec/lib/login_spec.rb

```diff
@@ -14,18 +14,18 @@
 
     it { should be_a Hash }
 
-    its([:sessionId]) do
+    its([:session_id]) do
```

Comment:
This restores a change previously made in https://github.com/Yesware/metaforce/commit/358f7a43bffeb04ecde0fc963f80a41b84d083c3